src/web_app/js/call.js

```diff
@@ -18,6 +18,40 @@
 
 var Call = function(params) {
   this.params_ = params;
+
```

Comment:
I don't like this much. You should not need to care about escape unescaping here.